src/nvim/fileio.c

```diff
@@ -2650,6 +2650,7 @@ buf_write(
    */
   if (!(append && *p_pm == NUL) && !filtering && perm >= 0 && dobackup) {
     FileInfo file_info;
+    const bool kNoPrependDot = false;
```

Comment:
`kNoPrependDot` is the format used for enum members. Just use `no_prepend_dot` instead.

Reply:
fixed, thanks!